.github/workflows/github_release.yml

```
@@ -87,43 +59,17 @@ jobs:
-          
-          # Renaming 32-bit APK and moving to safe staging directory
-          find $DEGOOGLED_PATH -name '*armeabi-v7a*signed.apk' -print0 | xargs -0 mv -t ./release-artifacts/
-          mv ./release-artifacts/*armeabi-v7a*signed.apk ./release-artifacts/app-degoogled-armeabi-v7a-release.apk
-          
-          echo "Prepared Degoogled APKs."
-          ls -la ./release-artifacts/
+          BUILD_TOOLS_VERSION: "35.0.0"
 
       - name: Create Release
         id: create_release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)